src/plugins/observables.js

```diff
@@ -10,27 +10,33 @@ export default {
 
     // Cast `decl` to `Observable`, since it passed the type guard.
     const style$ = (decl: Observable<{[string]: string}>)
-    const initialStyle = {}
+
+    // `style$` needs to be subscribed to before `rule` is created. Otherwise,
+    // any emissions that occur synchronously will be called on an unlinked
+    // `rule`, which will ignore them.
+    let rule: StyleRule
+    let initialStyle = {}
 
     // It can't be enumerable, otherwise it will be rendered.
     Object.defineProperty(initialStyle, 'isDynamic', {
       value: true,
       writable: false
     })
 
-    // We know `rule` is a `StyleRule`, and the other types don't have a
-    // `prop` method, so we must explicitly cast to `StyleRule`.
-    const rule = ((createRule(name, initialStyle, options): any): StyleRule)
-
     // TODO
     // Call `stream.subscribe()` returns a subscription, which should be explicitly
     // unsubscribed from when we know this sheet is no longer needed.
     style$.subscribe((style: JssStyle) => {
-      for (const prop in style) {
-        rule.prop(prop, style[prop])
+      if (rule) {
+        for (const prop in style) {
+          rule.prop(prop, style[prop])
+        }
       }
+      else if (Object.keys(style).length) initialStyle = style
```

Comment:
Trying to figure out the logic. Why do we postpone the rule creation and use style as initialStyle? 

Why not always create the rule first, before subscribe, then just update props with received value, it can go sync, it doesn't matter.

Reply:
I think I got it.

Reply:
If you call `rule.prop()` before linking happens, the style will never be set on `cssRule`, so we need somewhere to store the values that are emitted before linking.  `initialStyle` seems to be a good place to do that.

Reply:
I think you just discovered a completely different bug. Trying to fix it right now.